### ISSUE TITLE
Ikona nějak přestala fungovat

### ISSUE THREAD

**Issue by PetrDlouhy** [auto-mat/do-prace-na-kole#212]
Web nemá žádnou ikonu. Dříve tam byli ikony i pro mobily.

![Snímek obrazovky_2022-01-06_15-06-12](https://user-images.githubusercontent.com/156755/148395044-6bda87de-60a0-4b67-84d0-2210acc92571.png)

**Comment by timthelion** [comment]
No, to jsem dělal chybu při přechod na systém kde statické soubory se uloží do vlastní repozitařem se vlastní rychlejším buildem. Bohužel teď nevím jak to opravit, protože ten nový rychlejší build nahle přestal buildovat...

https://app.circleci.com/pipelines/github/auto-mat/do-prace-na-kole-frontend/31/workflows/eab574c5-17e6-4802-a49c-e474951a053d/jobs/37
